spec/classes/mcollective_spec.rb

```diff
@@ -145,6 +145,17 @@
             it { should contain_file('/usr/libexec/mcollective/refresh-mcollective-metadata').with_content(/File.rename\('\/tmp\/facts.new', '\/tmp\/facts'\)/) }
           end
         end
+
+        describe '#yaml_fact_cron' do
+          context 'default (true)' do
+            it { should contain_cron('refresh-mcollective-metadata') }
+          end
+
+          context 'false' do
+            let(:params) { { :server => false } }
```

Comment:
Why is this `:server` as opposed to `:yaml_fact_cron`?

I suspect that this is a simple copy/paste error, but then I really wonder how these ever passed.